node/src/accountant/payment_adjuster/non_unit_tests/mod.rs

```diff
@@ -378,14 +388,19 @@ fn pick_appropriate_cw_service_fee_balance(
     qualified_payables: &[QualifiedPayableAccount],
     accounts_count: usize,
 ) -> u128 {
-    // Value picked empirically
-    const COEFFICIENT: usize = 1000;
+    // Values picked empirically
+    const COEFFICIENT_A: usize = 1000;
+    const COEFFICIENT_B: usize = 2;
```

Comment:
I don't mind this code here, but these magic numbers causes problems in the future, please avoid them whenever possible.